doc/config_rsmeval.rst

```diff
@@ -47,20 +47,31 @@ The name for the column containing the human scores for each response. The value
 
     All responses with non-numeric values or zeros in either ``human_score_column`` or ``system_score_column`` will be automatically excluded from evaluation. You can use :ref:`exclude_zero_scores_eval` to keep responses with zero scores.
 
+.. _second_human_score_column_eval:
+
 second_human_score_column *(Optional)*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 The name for an optional column in the test data containing a second human score for each response. If specified, additional information about human-human agreement and degradation will be computed and included in the report. Note that this column must contain either numbers or be empty. Non-numeric values are *not* accepted. Note also that the :ref:`exclude_zero_scores_eval` option below will apply to this column too.
 
+.. note::
+
+    You do not need to have second human scores for *all* responses to use this option. The human-human agreement statistics will be computed as long as there is at least one response with numeric value in this column. The value for responses which do not have a second human score should be left blank. 
+
 .. _flag_column_eval:
 
 flag_column *(Optional)*
 ~~~~~~~~~~~~~~~~~~~~~~~~
-This field makes it possible to only use responses with particular values in a given column (e.g. only responses with a value of ``0`` in a column called ``ADVISORY``). The field takes a dictionary in Python format where the keys are the names of the columns and the values are lists of values for responses that will be evaluated. For example, a value of ``{"ADVISORY": 0}`` will mean that ``rsmtool`` will *only* use responses for which the ``ADVISORY`` column has the value 0. Defaults to ``None``.
+This field makes it possible to only use responses with particular values in a given column (e.g. only responses with a value of ``0`` in a column called ``ADVISORY``). The field takes a dictionary in Python format where the keys are the names of the columns and the values are lists of values for responses that will be evaluated. For example, a value of ``{"ADVISORY": 0}`` will mean that ``rsmeval`` will *only* use responses for which the ``ADVISORY`` column has the value 0. Defaults to ``None``.
 
 .. note::
 
     If  several conditions are specified (e.g., ``{"ADVISORY": 0, "ERROR": 0}``) only those responses which satisfy *all* the conditions will be selected for further analysis (in this example, these will be the responses where the ``ADVISORY`` column has a value of 0 *and* the ``ERROR`` column has a value of 0).
 
+.. note::
+
+    When comparing the values in the supplied dictionary and the data, ``rsmeval`` does not take into account the original type of the value. Thus ``1``, ``1.0``, ``"1"`` and ``"1.0"`` are treated as the same value. 
+
```

Comment:
>  When comparing the values in the supplied dictionary and the data, ``rsmeval`` does not take into account the original type of the value. Thus ``1``, ``1.0``, ``"1"`` and ``"1.0"`` are treated as the same value. 

When reading the values in the supplied dictionary, ``rsmeval`` treats numeric strings, floats and integers as the same value. Thus ``1``, ``1.0``, ``"1"`` and ``"1.0"`` are all treated as the 1.0.